src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
+spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False&\

```

Comment:
Here I suggest to revert back to the original version from master - the change doesn't seem to be needed for passing tests.